Jenkinsfile

```diff
@@ -57,7 +57,7 @@ pipeline {
         stage('checkout test project repo') {
             steps {
                 dir("$TEST_PROJECT_REPO_NAME"){
-                        checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/edx/edx-app-test.git']]])
+                        checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/openedx/edx-app-test.git']]])
```

Comment:
no need for this change now as another [PR](https://github.com/openedx/edx-app-android/pull/1699) is already related to this change is approved and will be merged soon.